unit_test/test_spdm_responder/digests.c

```diff
@@ -500,6 +500,101 @@ void libspdm_test_responder_digests_case9(void **state)
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_13 <<
+                                            SPDM_VERSION_NUMBER_SHIFT_BIT;
+    spdm_context->connection_info.connection_state =
+        LIBSPDM_CONNECTION_STATE_NEGOTIATED;
+    spdm_context->local_context.capability.flags = 0;
+    spdm_context->last_spdm_request_session_id_valid = false;
+    spdm_context->local_context.capability.flags |=
+        SPDM_GET_CAPABILITIES_RESPONSE_FLAGS_CERT_CAP;
+    spdm_context->connection_info.algorithm.base_hash_algo =
+        m_libspdm_use_hash_algo;
+
+    for (uint8_t index = 0; index < SPDM_MAX_SLOT_COUNT; index++) {
+        spdm_context->local_context.local_cert_chain_provision[index] =
+            &m_libspdm_local_certificate_chain[hash_size *index];
+        spdm_context->local_context.local_cert_chain_provision_size[index] = hash_size;
+    }
+
+    libspdm_set_mem(m_libspdm_local_certificate_chain,
+                    sizeof(m_libspdm_local_certificate_chain),
+                    (uint8_t)(0xFF));
+
+#if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT
+    spdm_context->transcript.message_m.buffer_size =
+        spdm_context->transcript.message_m.max_buffer_size;
+#endif
+    spdm_context->connection_info.multi_key_conn_rsp = true;
+    libspdm_reset_message_d(spdm_context);
+
+    response_size = sizeof(response);
+    status = libspdm_get_response_digests(spdm_context,
+                                          m_libspdm_get_digests_request2_size,
+                                          &m_libspdm_get_digests_request2,
+                                          &response_size, response);
+    assert_int_equal(status, LIBSPDM_STATUS_SUCCESS);
+    assert_int_equal(response_size,
+                     sizeof(spdm_digest_response_t) +
+                     (hash_size + additional_size) *
+                     slot_count);
+
+    spdm_response = (void *)response;
+    assert_int_equal(spdm_response->header.request_response_code,
+                     SPDM_DIGESTS);
+    assert_int_equal(spdm_context->transcript.message_d.buffer_size,
+                     sizeof(spdm_digest_response_t) + (hash_size + additional_size) * slot_count);
+
+    digest = (void *)(spdm_response + 1);
+    libspdm_zero_mem (digest, hash_size * slot_count);
+    key_pair_id = (spdm_key_pair_id_t *)((uint8_t *)digest + (hash_size * slot_count));
+    cert_info = (spdm_certificate_info_t *)((uint8_t *)key_pair_id +
+                                            sizeof(spdm_key_pair_id_t) * slot_count);
+    key_usage_bit_mask = (spdm_key_usage_bit_mask_t *)((uint8_t *)cert_info +
+                                                       sizeof(spdm_certificate_info_t) *
+                                                       slot_count);
+    for (uint8_t index = 0; index < SPDM_MAX_SLOT_COUNT; index++) {
+        assert_memory_equal((void *)&key_pair_id[index],
+                            (void *)&spdm_context->local_context.local_key_pair_id[index],
```

Comment:
`spdm_context->local_context.local_key_pair_id[index]`, what is its value? random value?